CHANGELOG.md

```diff
@@ -33,6 +33,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Fixed urllib3 instrumentation example in instrumentation documentation
```

Comment:
Please move this entry to the Unreleased section

Reply:
Actually, we don't need a changelog entry for this PR.